commit-graph.c

```diff
@@ -1986,14 +1986,18 @@ int verify_commit_graph(struct repository *r, struct commit_graph *g, int flags)
 	if (verify_commit_graph_error & ~VERIFY_COMMIT_GRAPH_ERROR_HASH)
 		return verify_commit_graph_error;
 
-	progress = start_progress(_("Verifying commits in commit graph"),
-				  g->num_commits);
+	if (flags & COMMIT_GRAPH_PROGRESS)
+		progress = start_progress(_("Verifying commits in commit graph"),
+					g->num_commits);
+
 	for (i = 0; i < g->num_commits; i++) {
 		struct commit *graph_commit, *odb_commit;
 		struct commit_list *graph_parents, *odb_parents;
 		uint32_t max_generation = 0;
 
-		display_progress(progress, i + 1);
+		if (flags & COMMIT_GRAPH_PROGRESS)
```

Comment:
Sorry, @garimasi514, for not noticing this earlier: `display_progress()` and `stop_progress()` know how to handle a `NULL` progress pointer. It's already initialized to `NULL` above.

Reply:
cc: @wilbaker